tests/em/fdem/forward/test_FDEM_analytic_Dipoles.py

```diff
@@ -0,0 +1,1494 @@
+            plt.plot(self.r, hz_num.imag, 'o', self.r, hz_numTest.imag, 'd', self.r, hza.imag, linewidth=2)
+            plt.grid(which='both')
+            plt.title('H_z Imag')
+            plt.legend(['Num','Ana'],bbox_to_anchor=(1.5,0.5))
+            plt.xlabel('r (m)')
+
+            plt.legend(['Num', 'NumTest', 'Ana'],bbox_to_anchor=(1.5,0.5))
+            plt.tight_layout()
+
+             # Plot B
+            fig, ax = plt.subplots(2,3, figsize=(20,10))
+            plt.subplot(231)
+            plt.plot(self.r, bx_num.real, 'o', self.r, bx_numTest.real, 'd', self.r, bxa.real, linewidth=2)
+            plt.grid(which='both')
+            plt.title('B_x Real')
+            plt.xlabel('r (m)')
+
+            plt.subplot(234)
+            plt.plot(self.r, bx_num.imag, 'o', self.r, bx_numTest.imag, 'd', self.r, bxa.imag, linewidth=2)
+            plt.grid(which='both')
+            plt.title('B_x Imag')
+            plt.xlabel('r (m)')
+
+            plt.subplot(232)
+            plt.plot(self.r, btheta_num.real, 'o', self.r, btheta_numTest.real, 'd', self.r, bya.real, linewidth=2)
+            plt.grid(which='both')
+            plt.title('B_y Real')
+            plt.xlabel('r (m)')
+
+            plt.subplot(235)
+            plt.plot(self.r, btheta_num.imag, 'o', self.r, btheta_numTest.imag, 'd', self.r, bya.imag, linewidth=2)
+            plt.grid(which='both')
+            plt.title('B_y Imag')
+            plt.xlabel('r (m)')
+
+            plt.subplot(233)
+            plt.plot(self.r, bz_num.real, 'o', self.r, bz_numTest.real, 'd', self.r, bza.real, linewidth=2)
+            plt.grid(which='both')
+            plt.title('B_z Real')
+            plt.xlabel('r (m)')
+
+            plt.subplot(236)
+            plt.plot(self.r, bz_num.imag, 'o', self.r, bz_numTest.imag, 'd', self.r, bza.imag, linewidth=2)
+            plt.grid(which='both')
+            plt.title('B_z Imag')
+            plt.legend(['Num','Ana'],bbox_to_anchor=(1.5,0.5))
+            plt.xlabel('r (m)')
+
+            plt.legend(['Num', 'NumTest', 'Ana'],bbox_to_anchor=(1.5,0.5))
+            plt.tight_layout()
+
+
+class FDEM_analytic_DipoleTests_3DMesh(unittest.TestCase):
+
+    def setUp(self):
+
+        # Define model parameters
+        self.sigmaback = 1.
+        # mu = mu_0*(1+kappa)
+        self.kappa = 1.
+
+        # Create 3D mesh
+        npad = 5
+        self.csx, self.ncx, self.npadx = 5, 30, npad
+        self.csy, self.ncy, self.npady = 5, 30, npad
+        self.csz, self.ncz, self.npadz = 5, 30, npad
+        self.hx = Utils.meshTensor([(self.csx, self.npadx, -1.3), (self.csx, self.ncx), (self.csx, self.npadx, 1.3)])
+        self.hy = Utils.meshTensor([(self.csy, self.npady, -1.3), (self.csy, self.ncy), (self.csy, self.npady, 1.3)])
+        self.hz = Utils.meshTensor([(self.csz, self.npadz, -1.3), (self.csz, self.ncz), (self.csz, self.npadz, 1.3)])
+        self.mesh = Mesh.TensorMesh([self.hx, self.hy, self.hz], 'CCC')
```

Comment:
This is a reasonably large mesh (40 x 40 x 40). Any chance we can reduce that so this runs faster? eg in the analytics tests, we try to keep them to 20 x 20 x 20 so that the tests are almost real-time